python/ray/dashboard/modules/usage_stats/usage_stats_head.py

```diff
@@ -67,7 +67,7 @@ def _check_grafana_running(self):
 
         grafana_running = False
         try:
-            resp = requests.get(f"{self._dashboard_url_base}/api/grafana_health")
+            resp = requests.get(f"{self._dashboard_url_base}")
```

Comment:
is this path guaranteed to return a 200 when grafana is running?

Reply:
It's a wrong episode, thanks for correcting.